docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-okta-saml.md

```diff
@@ -60,9 +62,41 @@ Setting | Value
 
 :::note SAML Provider Caveats:
 
-- SAML Protocol does not support search or lookup for users or groups. Therefore, there is no validation on users or groups when adding them to Rancher.
+If you configure Okta without OpenLDAP, the following caveats apply due to the fact that SAML Protocol does not support search or lookup for users or groups.
+
+- There is no validation on users or groups when assigning permissions to them in Rancher.
```

Comment:
The phrase "use OpenLDAP as your Okta backend" ends up being extra confusing, but mostly because Okta itself is more flexible. It can either use its own directory layered with an LDAP _interface_, or it can be set up as a wrapper around some backing identity provider, which may either _be_ OpenLDAP, or provide its own separate LDAP interface. Rancher doesn't especially care how the various IdP bits link together, it just needs SAML details and LDAP details that map to the same users.

Since all of these varied flows are supported, we should try to avoid suggesting that a specific backend configuration is required.

Reply:
Ok, I wasn't aware of the distinction. You can ignore the suggestion.